pkg/status/status.go

```diff
@@ -10,18 +10,21 @@ import (
 
 	"github.com/redhat-developer/app-services-cli/pkg/connection"
 	"github.com/redhat-developer/app-services-cli/pkg/kafka/kafkaerr"
+	"github.com/redhat-developer/app-services-cli/pkg/serviceregistry"
 	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
 
 	"github.com/openconfig/goyang/pkg/indent"
 	"github.com/redhat-developer/app-services-cli/internal/config"
 	kas "github.com/redhat-developer/app-services-cli/pkg/api/kas"
 	"github.com/redhat-developer/app-services-cli/pkg/logging"
+	srsmgmtv1 "github.com/redhat-developer/app-services-sdk-go/registrymgmt/apiv1/client"
 )
 
 const tagTitle = "title"
 
 type Status struct {
-	Kafka *KafkaStatus `json:"kafka,omitempty" title:"Kafka"`
+	Kafka    *KafkaStatus    `json:"kafka,omitempty" title:"Kafka"`
+	Registry *RegistryStatus `json:"registry,omitempty" title:"Registry"`
```

Comment:
Should we call this "Service Registry"?

```suggestion
	Registry *RegistryStatus `json:"registry,omitempty" title:"Service Registry"`
```